### ISSUE TITLE
Guardian's Stone crashing the game

### ISSUE THREAD

**Issue by Franzoli-GitHub** [TrashboxBobylev/Summoning-Pixel-Dungeon#47]
Guardian's stone crashing the game when it is dropped, thrown,unequipped or transmuted

**Comment by TrashboxBobylev** [comment]
Do you have any crash log?

**Comment by Franzoli-GitHub** [comment]
***** UCE HANDLER Library 
***** by Rohit Surwase 

***** DEVICE INFO 
Brand: HONOR
Device: HWRNE
Model: RNE-L22
Manufacturer: HUAWEI
Product: RNE-L22
SDK: 26
Release: 8.0.0

***** APP INFO 
Version: 1.2.2n-27760e4
Installed On: 2022-09-29 20:21:08
Updated On: 2022-10-01 09:35:16
Current Date: 2022-10-01 21:04:35

***** ERROR LOG 
java.lang.NullPointerException: Attempt to invoke virtual method 'void com.shatteredpixel.shatteredpixeldungeon.actors.buffs.Buff.detach()' on a null object reference
	at com.shatteredpixel.shatteredpixeldungeon.items.artifacts.Artifact.doUnequip(Artifact.java:96)
	at com.shatteredpixel.shatteredpixeldungeon.items.artifacts.ringartifacts.HeavyFlail.doUnequip(HeavyFlail.java:238)
	at com.shatteredpixel.shatteredpixeldungeon.items.EquipableItem.doUnequip(EquipableItem.java:146)
	at com.shatteredpixel.shatteredpixeldungeon.items.EquipableItem.execute(EquipableItem.java:86)
	at com.shatteredpixel.shatteredpixeldungeon.items.artifacts.ringartifacts.HeavyFlail.execute(HeavyFlail.java:165)
	at com.shatteredpixel.shatteredpixeldungeon.windows.WndUseItem$1.onClick(WndUseItem.java:64)
	at com.watabou.noosa.ui.Button$1.onClick(Button.java:63)
	at com.watabou.noosa.PointerArea.onSignal(PointerArea.java:84)
	at com.watabou.noosa.PointerArea.onSignal(PointerArea.java:30)
	at com.watabou.utils.Signal.dispatch(Signal.java:78)
	at com.watabou.input.PointerEvent.processPointerEvents(PointerEvent.java:96)
	at com.watabou.input.InputHandler.processAllEvents(InputHandler.java:81)
	at com.watabou.noosa.Game.update(Game.java:266)
	at com.watabou.noosa.Game.step(Game.java:233)
	at com.watabou.noosa.Game.render(Game.java:161)
	at com.badlogic.gdx.backends.android.AndroidGraphics.onDrawFrame(AndroidGraphics.java:471)
	at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1609)
	at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1293)



***** END OF LOG *****
